iiblib/iibclient.py

```diff
@@ -504,10 +504,12 @@ def add_bundles(
         post_data = {
             "from_index": index_image,
             "binary_image": binary_image,
-            "bundles": bundles,
```

Comment:
why not do `bundles or []`

Reply:
This is a POST request. Why input an unused parameter? Also, acceptance criteria in CLOUDDST-2204 imply that bundles should not be used as a parameter instead of being an empty list.

Reply:
Well, I just looked at the IIB code and it looks like it will default to an empty list if the parameter is not provided. So I guess it should be fine.